tests/subsys/app_event_manager/src/modules/test_name_style_sorting.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/ztest.h>
+
+#include "test_events.h"
+#include "name_style_events.h"
+
+#define MODULE test_name_style_sorting
+
+enum test_name_style_sorting_step {
+	TESTING_EVENT_NAME_STYLE,
+	TESTING_EVENT_NAME_STYLE_BREAK,
+	TESTING_EVENT_NAME_STYLE_END,
+};
+
+static enum test_name_style_sorting_step test_current_stage;
+static int test_name_style_cnt;
+
+
+/**
+ * @brief Assert that @a e event subscribers does not start inside @a er subscribers range
+ *
```

Comment:
Do we need 2 empty lines here?

Reply:
No - we don't. I can see we are using two styles here in the codebase - one uses two empty lines after all the module global variables other does not. I have removed this anyway. No strong preference on my side.